src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -124,87 +115,79 @@ protected static String getPopulatedEmptyStringErrorMessage(String messageTempla
      */
 
     protected static void assertTrue(boolean condition) {
-        AssertJUnit.assertTrue(condition);
+        Assertions.assertTrue(condition);
```

Comment:
This comment is not related to this PR: use a static import so that `Assertions.` part can be omitted?

Reply:
Not possible; immediate name clash.